modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -231,24 +238,51 @@ $ oc get configmap kubevirt-vm-latency-checkup -n kiagnose -o yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kubevirt-vm-latency-checkup
-  namespace: kiagnose
-...
+  name: kubevirt-vm-latency-checkup-config
+  namespace: <target_namespace>
+data:
+  spec.image: registry.redhat.io/container-native-virtualization/vm-network-latency-checkup:v4.12.0
+  spec.timeout: 5m
+  spec.serviceAccountName: vm-latency-checkup-sa
+  spec.param.network_attachment_definition_namespace: <target_namespace>
+  spec.param.network_attachment_definition_name: "bridge-network"
+  spec.param.max_desired_latency_milliseconds: "10"
+  spec.param.sample_duration_seconds: "5"
+  spec.param.source_node: "worker1"
+  spec.param.target_node: "worker2"
   status.succeeded: "true"
   status.failureReason: ""
-  status.result.minLatencyNanoSec: 2000
-  status.result.maxLatencyNanoSec: 3000
-  status.result.avgLatencyNanoSec: 2500
-  status.results.measurementDurationSec: 300
-...
+  status.completionTimestamp: "2022-01-01T09:00:00Z"
+  status.startTimestamp: "2022-01-01T09:00:07Z"
+  status.result.avgLatencyNanoSec: "177000"
+  status.result.maxLatencyNanoSec: "244000" <1>
+  status.result.measurementDurationSec: "5"
+  status.result.minLatencyNanoSec: "135000"
+  status.result.sourceNode: "worker1"
+  status.result.targetNode: "worker2"
 ----
+<1> The maximum measured latency in nanoseconds.
 
-. Delete the framework and checkup resources that you previously created. This includes the job, config map, cluster role, and framework manifest files.
+. Optional: To view the detailed job log in case of checkup failure, use the following command:
 +
-[NOTE]
-====
-Do not delete the framework and cluster role manifest files if you plan to run another checkup.
-====
+[source,terminal]
+----
+$ oc logs job.batch/kubevirt-vm-latency-checkup -n kiagnose
```

Comment:
This will not work due to https://bugzilla.redhat.com/show_bug.cgi?id=2134805

To fix it, we need to extend the cluster role on line 96 with https://github.com/kiagnose/kiagnose/pull/174/files#diff-c3f7b88b18be754cbf248f01b8d341fa013e99a7f2d77924b798c26a733f4e66R44-R51

Reply:
@sjhala-ccs I talked with Orel. With the newly added 4.13->4.12 epic, this BZ becomes irrelevant. I will close the BZ and we will not need to extend the RBAC in this PR.

Even though the changes in this PR will need to be reworked while covering the new epic, I would still recommend merging this PR as it covers some new fields in the latency checkup that will remain relevant.